src/main/java/frc/robot/commands/ShooterModeCycleDown.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShooterModeCycleDown extends CommandBase {
+  /** Creates a new ShooterModeCycleDown. */
+  private ShooterSubsystem m_ShooterSubsystem;
+
+  public ShooterModeCycleDown(ShooterSubsystem shooterSubsystem) {
+    addRequirements(shooterSubsystem);
+    m_ShooterSubsystem = shooterSubsystem;
+
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_ShooterSubsystem.cycleAimModeDown();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```

Comment:
Using the whenPressed trigger means you have to determine end conditions or it'll keep running. In this case you probably just want to change the return value to true since you want to initialize and then end.  

You can save time and code by converting this into an [InstantCommand](https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html#instantcommand)

Reply:
Converting these to InstantCommands.